controller/users_blackbox_test.go

```diff
@@ -153,3 +265,18 @@ func assertUser(t *testing.T, actual *app.IdentityData, expectedUser account.Use
 	assert.Equal(t, expectedUser.ImageURL, *actual.Attributes.ImageURL)
 	assert.Equal(t, expectedUser.Email, *actual.Attributes.Email)
 }
+
+func createUpdateUsersPayload(email *string, fullName *string, bio *string, imageURL *string, profileURL *string) *app.UpdateUsersPayload {
```

Comment:
Suggestion: All the input arguments are of the same type so you could do the following `email, fullName, bio, imageURL, profileURL *string)`